State.py

```diff
@@ -80,6 +80,20 @@ def save_jobs(self):
         else:
             print("All permitted jobs have been created, please come back later.\n")
             return False
+        
+
+    def saved_jobs(self):
```

Comment:
I think this is supposed to be "save_jobs" which is what is called in another file

Reply:
no it is not, save_jobs does something else